tensorflow_quantum/core/serialize/serializer.py

```diff
@@ -970,3 +971,72 @@ def _process_pauli_type(char):
     if char == 'Y':
         return cirq.Y
     raise ValueError("Invalid pauli type.")
+
+
+def serialize_projectorsum(projectorsum):
+    """Constructs a projector_sum proto from `cirq.ProjectorSum`.
+
+    Args:
+        projectorsum: A `cirq.ProjectorSum` or `cirq.ProjectorString` object.
+
+    Returns:
+        A projector_sum proto object.
+    """
+    if isinstance(projectorsum, cirq.ProjectorString):
+        projectorsum = cirq.ProjectorSum.from_pauli_strings(projectorsum)
+
+    if not isinstance(projectorsum, cirq.ProjectorSum):
+        raise TypeError("serialize requires a cirq.ProjectorSum object."
+                        " Given: " + str(type(projectorsum)))
+
+    if any(not isinstance(qubit, cirq.GridQubit)
+           for qubit in projectorsum.qubits):
+        raise ValueError("Attempted to serialize a paulisum that doesn't use "
+                         "only cirq.GridQubits.")
+
+    projectorsum_proto = projector_sum_pb2.ProjectorSum()
+    for term in projectorsum:
+        projectorterm_proto = projector_sum_pb2.ProjectorTerm()
+
+        projectorterm_proto.coefficient_real = term.coefficient.real
+        projectorterm_proto.coefficient_imag = term.coefficient.imag
+        for qubit, basis_state in sorted(
+                term.projector_dict.items()):  # sort to keep qubits ordered
+            if basis_state:
+                projectorterm_proto.projector_dict.add(
+                    qubit_id=op_serializer.qubit_to_proto(qubit),
+                    basis_state=basis_state)
+            else:
+                projectorterm_proto.projector_dict.add(
+                    qubit_id=op_serializer.qubit_to_proto(qubit))
```

Comment:
Nit: Could we not do:
```python3
projectorterm_proto.projector_dict.add(
                    qubit_id=op_serializer.qubit_to_proto(qubit),
                    basis_state=basis_state)
```
In the case of basis_state being true or false ?

Reply:
Done.